scripts/useReactVersion.mjs

```diff
@@ -37,7 +37,9 @@ async function main(version) {
     return;
   }
 
-  const packageJsonPath = path.resolve(getWorkspaceRoot(), 'package.json');
+  const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+  const workspaceRoot = path.resolve(currentDirectory, '..');
```

Comment:
Can't this script just default to `process.cwd()` and add the script to root package.jsons scripts. Then it will always run with cwd of the root, regardless of whether it's executed from a subfolder.

Reply:
My personal preference is not to include everything in package.json scripts to avoid cluttering them, especially if the script is only used by the CI.